cross/build-rootfs.sh

```diff
@@ -52,6 +52,9 @@ for i in "$@" ; do
         arm64)
             __BuildArch=arm64
             __UbuntuArch=arm64
+            if [ "$__LLDB_Package" = "lldb-3.6-dev" ]; then
```

Comment:
I'm not sure I follow some of this logic.  Won't this be overwritten if someone specifies `lldb3.6` on the command-line after `arm64`?

ex. `build-rootfs.sh arm64 lldb3.6`.

Would it be better to just change the caller to specify `lldb3.9`?

Reply:
Or, after processing all the command line args, then check if `__BuildArch == arm64`, set `__LLDB_PACKAGE=lldb-3.9-dev`.

---
In reply to: [173321011](https://github.com/dotnet/core-setup/pull/3801#discussion_r173321011) [](ancestors = 173321011)

Reply:
Will change caller